corehq/apps/app_manager/static/app_manager/js/forms/advanced/case_config_ui.js

```diff
@@ -24,6 +24,7 @@ hqDefine('app_manager/js/forms/advanced/case_config_ui', function () {
                 $('.property-description').closest('.read-only').popover({
                     'trigger': 'hover',
                     'placement': 'bottom',
+                    'sanitize': false,
```

Comment:
Is this safe, given that the property description is user input?

Reply:
updated